web/packages/teleport/src/Roles/RoleEditor/StandardEditor/standardmodel.ts

```diff
@@ -1321,7 +1406,9 @@ export function roleEditorModelToRole(roleModel: RoleEditorModel): Role {
   if (roleModel.rules.length > 0) {
     role.spec.allow.rules = roleModel.rules.map(role => ({
       resources: role.resources.map(r => r.value),
-      verbs: role.verbs.map(v => v.value),
+      verbs: role.allVerbs
+        ? ['*']
+        : role.verbs.filter(v => v.checked).map(v => v.verb),
```

Comment:
Why do we need to do this? 
The resulting value/interpretation is same but, from role management perspective, a wildcard value is usually discouraged and the system should not auto-print that unless user explicitly configures a wildcard value.

Reply:
> unless user explicitly configures a wildcard value

i believe that is the intention of the "all" value, to say "i want a wildcard"

Reply:
`role.allVerbs` is tied to the separate "all verbs" checkbox, which overrides the other model variables. The specific verbs are not converted to values in the role, since they are redundant in presence of the wildcard. Perhaps the code is not clear enough and I should add a comment?

Reply:
Code looks good. The checkbox "all" selection are usually implementation detail rather than underlying value so I questioned why pick `*` instead of deducing verbs. I just looked at the video and saw we do have `All (*)` label so that is good too. 

May I suggest to rename the all checkbox label as `All (Wildcard verb "*")` instead of just `All (*)`. Its a bit wordy but clear on the final role verb.

Reply:
Yeah, I think it's a valid point. I'll make the checkbox label a bit more explicit here, especially that in other contexts, we use the asterisk as a symbol of a mandatory input.